go/client/chat_svc_handler.go

```diff
@@ -184,16 +184,48 @@ func (c *chatServiceHandler) SendV1(ctx context.Context, opts sendOptionsV1) Rep
 
 // DeleteV1 implements ChatServiceHandler.DeleteV1.
 func (c *chatServiceHandler) DeleteV1(ctx context.Context, opts deleteOptionsV1) Reply {
-	convID, err := chat1.MakeConvID(opts.ConversationID)
+	client, err := GetChatLocalClient(c.G())
+	if err != nil {
+		return c.errReply(err)
+	}
+	convID, _, err := c.resolveAPIConvID(ctx, opts.ConversationID, opts.Channel)
 	if err != nil {
 		return c.errReply(fmt.Errorf("invalid conv ID: %s", opts.ConversationID))
 	}
+
+	// Fetch any edits that exist for this message.
```

Comment:
Ok, as just discussed, I'll go ahead and add the `supersedes` field to the DB so that we can run this query server side.